bench/wtperf/wtperf.c

```diff
@@ -1810,8 +1810,8 @@ create_tables(CONFIG *cfg)
 
 	for (i = 0; i < cfg->table_count; i++) {
 		if (cfg->log_partial && i > 0) {
-			if (((ret = session->create(session,
-			    cfg->uris[i], cfg->partial_config)) != 0)) {
+			if (((ret = session->create(session, cfg->uris[i],
+			    (const char *)cfg->partial_config)) != 0)) {
```

Comment:
@sueloverso, I'm not following the need for the `(const char *)` cast here, or above?